### PR TITLE
reboot: add pt_BR translation

### PR DESCRIPTION
**Pull request by felixstenian**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/reboot.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# reboot
+
+> Reinicie o sistema.
+> Mais Informações: <https://manned.org/reboot.8>.
+
+- Reinicie o sistema:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:6]:
```suggestion
- Reinicia o sistema:
```

File: pages.pt_BR/linux/reboot.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Desligue o sistema (igual a `poweroff`):
+
+`reboot --poweroff`
+
+- Suspenda o sistema (igual a `halt`):
+
+`reboot --halt`
+
+- Reinicie imediatamente sem entrar em contato com o gerente do sistema:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:18]:
```suggestion
- Reinicia imediatamente sem entrar em contato com o gerente do sistema:
```

File: pages.pt_BR/linux/reboot.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`reboot --force`
+
+- Escreva a entrada wtmp shutdown sem reinicializar o sistema:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:22]:
```suggestion
- Escreve a entrada wtmp shutdown sem reinicializar o sistema:
```

File: pages.pt_BR/linux/reboot.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Reinicie o sistema:
+
+`reboot`
+
+- Desligue o sistema (igual a `poweroff`):
+
+`reboot --poweroff`
+
+- Suspenda o sistema (igual a `halt`):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:14]:
```suggestion
- Suspende o sistema (igual a `halt`):
```

File: pages.pt_BR/linux/reboot.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# reboot
+
+> Reinicie o sistema.
+> Mais Informações: <https://manned.org/reboot.8>.
+
+- Reinicie o sistema:
+
+`reboot`
+
+- Desligue o sistema (igual a `poweroff`):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:10]: present indicative
```suggestion
- Desliga o sistema (igual a `poweroff`):
```

File: pages.pt_BR/linux/reboot.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# reboot
+
+> Reinicie o sistema.
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:3]:
```suggestion
> Reinicia o sistema.
```

File: pages.pt_BR/linux/reboot.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+> Mais Informações: <https://manned.org/reboot.8>.
```
Comment (navarroaxel) [inline pages.pt_BR/linux/reboot.md:4]:
```suggestion
> Mais informações: <https://manned.org/reboot.8>.
```

**Review by navarroaxel — COMMENTED** [review]
Thanks @felixstenian, I left some comments.

**Comment by felixstenian** [comment]
Thanks @navarroaxel for the comments.

**Review by navarroaxel — APPROVED** [review]
LGTM.
Remember that you can use this GitHub feature <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request> to apply all the suggestions in 1 commit.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**